debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/sink/MongoDbChangeEventSink.java

```diff
@@ -57,8 +57,8 @@ public void close() {
         }
     }
 
-    public Optional<CollectionId> getCollectionId(String collectionName) {
-        return Optional.of(new CollectionId(collectionName));
+    public CollectionId getCollectionId(String collectionName) {
```

Comment:
Why the change from `Optional`? I amnot against it I' like to know the motivation.